src/Modules/UngradedModules.jl

```diff
@@ -1539,7 +1543,7 @@ function is_canonically_isomorphic(M::SubModuleOfFreeModule, N::SubModuleOfFreeM
   return SubModuleOfFreeModule(G, [f(v) for v in gens(M)]) == N
 end
 
-#+(M::SubModuleOfFreeModule, N::SubModuleOfFreeModule) = sum(M, N)
+Base.:+(M::SubModuleOfFreeModule, N::SubModuleOfFreeModule) = sum(M, N)
```

Comment:
I wonder why this was commented out (@jankoboehm do you know)? According to Codecov, nothing in our test suite seems to call it right now, making me wary if there is something is subtly wrong here?